thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/controller.js

```diff
@@ -281,190 +275,144 @@ export default Controller.extend({
-      }
-    });
+  _resetLocalFilters() {
+    let alertFilters = {};
+    const newFilterBlocksLocal = _.cloneDeep(get(this, 'initialFiltersLocal'));
 
-    // If status filter is present, we re-build the results array to contain only active alerts, inactive alerts, or both.
-    if (filters.status) {
-      const concatStatus = filters.status.length ? filters.status.join().toLowerCase() : 'active';
-      const requireAll = filters.status.includes('Active') && filters.status.includes('Inactive');
-      const alertsByState = {
-        active: filteredAlerts.filter(alert => alert.active),
-        inactive: filteredAlerts.filter(alert => !alert.active)
-      };
-      filteredAlerts = requireAll ? [ ...alertsByState.active, ...alertsByState.inactive ] : alertsByState[concatStatus];
-    }
+    // Do not highlight any of the primary filters
+    Object.assign(alertFilters, { primary: 'none' });
 
-    return filteredAlerts;
+    // Reset local (secondary) filters, and set select fields to 'disabled'
+    setProperties(this, {
+      filterBlocksLocal: newFilterBlocksLocal,
+      resetFiltersLocal: moment().valueOf(),
+      allowFilterSummary: false,
+      alertFilters
+    });
   },
 
+  _getAlerts: task (function * (params) {
+    const subscriptionGroups = this.get('subscriptionGroupConfig');
+    const alerts = yield fetch(yamlAPI.getPaginatedAlertsUrl(params)).then(checkStatus);
+    const enrichedAlerts = enrichAlertResponseObject((alerts || {}).elements, subscriptionGroups);
+    set(this, 'originalAlerts', enrichedAlerts);
+    set(this, 'totalNumberOfAlerts', (alerts || {}).count);
+    // Reset secondary filters
+    setProperties(this, {
+      filterBlocksLocal: _.cloneDeep(get(this, 'initialFiltersLocal')),
+      resetFiltersLocal: moment().valueOf()
+    });
+  }).keepLatest(),
+
   /**
-   * Simply select the appropriate set of alerts called for by primary filter
-   * @method _processPrimaryFilters
-   * @param {Array} originalAlerts - array of all alerts from model
-   * @param {Object} primaryFilter - filter key/value for primary filter selections
-   * @returns {undefined}
-   * @private
+   * Ember concurrency task that triggers alert names autocomplete
    */
-  _processPrimaryFilters (originalAlerts, primaryFilter) {
-    const { ownedAlerts, subscribedAlerts } = getProperties(this, 'ownedAlerts', 'subscribedAlerts');
+  _searchAlertNames: task(function* (text) {
+    yield timeout(1000);
+    return fetch(autocompleteAPI.alertByName(text))
+      .then(checkStatus);
+  }),
+
+  _fetchAlerts() {
+    const paramsForAlerts = get(this, 'paramsForAlerts');
+    return this.get('_getAlerts').perform(paramsForAlerts);
+  },
 
-    let newAlerts = [];
+  _handlePrimaryFilter(primaryFilter, paramsForAlerts) {
     switch(primaryFilter) {
       case 'Alerts I subscribe to': {
-        newAlerts = subscribedAlerts;
+        const user = this.get('user');
+        paramsForAlerts['subscribedBy'] = user;
```

Comment:
One little catch here. The user string is in the format of 'username@linkedin.biz', but the email written in the subscription group is stored as 'username@linkedin.com'. So directly searching the user as a string won't get any result. Maybe change it to send 'username@linkedin.com' or 'userName' directly?

Reply:
I added a split for grabbing the username.  The token is configured in environment.js, with the default being ' '.